pkg/interface/src/views/apps/chat/components/ChatInput.tsx

```diff
@@ -187,12 +188,16 @@ export default class ChatInput extends Component<ChatInputProps, ChatInputState>
         />;
 
     return (
-      <div className={
-             "cf items-center flex black white-d bt b--gray4 b--gray1-d bg-white"  +
-             "bg-gray0-d relative"
-           }
-           style={{ flexGrow: 1 }}
-           >
+      <Row
+        alignItems='center'
+        position='relative'
```

Comment:
`position='relative'` is the default now on block level indigo components I believe